src/cert/chain.rs

```diff
@@ -0,0 +1,238 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Cert chains.
+
+use core::num::NonZeroUsize;
+
+use arrayvec::ArrayVec;
+
+use crate::cert::Cert;
+use crate::cert::CertFormat;
+use crate::cert::Error;
+use crate::crypto::sig;
+use crate::io;
```

Comment:
This `use crate::io;` is unused and is making CI fail.

Reply:
Embarassing...